policyengine_us/variables/gov/states/va/tax/income/va_rebate.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class va_rebate(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Virginia rebate"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://budget.lis.virginia.gov/item/2023/2/HB6001/Introduced/3/3-5.28/"
+    defined_for = StateCode.VA
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.va.tax.income.rebate
+        if p.availability:
+            filing_status = tax_unit("filing_status", period)
+            return p.amount[filing_status]
+        return 0
```

Comment:
```suggestion
    def formula(tax_unit, period, parameters):
        p = parameters(period).gov.states.va.tax.income.rebate
        filing_status = tax_unit("filing_status", period)
        return p.amount[filing_status]
```

Reply:
I made a mental mistake, if we add it to the 2023 non refundable credits then we by design dont need the if condition

Reply:
but then the `va_rebate.yaml` will report error with 2022 test cases

Reply:
Yes but we never pass it through the 2022 simulation if we only include it in the 2023 period non_refundable file

Reply:
so then should I delete 2022 test case in `va_rebate.yaml`?

Reply:
Yes please